src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -273,26 +267,29 @@ void IIssueConsumer.Accept(string path, IEnumerable<IAnalysisIssue> issues)
                 return;
             }
 
-            var filteredIssues = RemoveSuppressedIssues(issues);
-
             // See bug #1487: we should be creating the issue markers using the text snapshot
             // from when the analysis was triggered, not the current one.
-            var newMarkers = filteredIssues.Where(IsValidIssueTextRange).Select(CreateIssueMarker);
-            UpdateIssues(newMarkers);
+            var newMarkers = issues
+                .Where(IsValidIssueTextRange)
```

Comment:
I re-ordered the processing slightly so that out-of-range issues are still removed before the new `IssueMarker` instances are created.
However, I don't think this matters too much; the ordering will change again when we fix #1487, and we might be able to drop `IsValidIssueTextRange`.